CDC-Data-Reconciliation-Backend/config.json

```diff
@@ -3,5 +3,6 @@
   "server": "localhost",
   "database": "NBS_ODSE",
   "username": "sa",
-  "password": "Gt24ye??"
+  "password": "Gt24ye??",
```

Comment:
a bit of a nit: but I would rename this field to db_password

Reply:
Seems like these should be fixed in a different PR; outside of the scope of this change

Reply:
Since it would need to be changed in other places in the code also